azurecompute-arm/src/main/java/org/jclouds/azurecomputearm/compute/functions/DeploymentToNodeMetadata.java

```diff
@@ -35,48 +35,29 @@
-           put(Deployment.InstanceStatus.DELETING_VM, NodeMetadata.Status.PENDING).
-           put(Deployment.InstanceStatus.FAILED_STARTING_ROLE, NodeMetadata.Status.ERROR).
-           put(Deployment.InstanceStatus.FAILED_STARTING_VM, NodeMetadata.Status.ERROR).
-           put(Deployment.InstanceStatus.PREPARING, NodeMetadata.Status.PENDING).
-           put(Deployment.InstanceStatus.PROVISIONING, NodeMetadata.Status.PENDING).
-           put(Deployment.InstanceStatus.READY_ROLE, NodeMetadata.Status.RUNNING).
-           put(Deployment.InstanceStatus.RESTARTING_ROLE, NodeMetadata.Status.PENDING).
-           put(Deployment.InstanceStatus.STARTING_ROLE, NodeMetadata.Status.PENDING).
-           put(Deployment.InstanceStatus.STARTING_VM, NodeMetadata.Status.PENDING).
-           put(Deployment.InstanceStatus.STOPPED_DEALLOCATED, NodeMetadata.Status.SUSPENDED).
-           put(Deployment.InstanceStatus.STOPPED_VM, NodeMetadata.Status.SUSPENDED).
-           put(Deployment.InstanceStatus.STOPPING_ROLE, NodeMetadata.Status.PENDING).
-           put(Deployment.InstanceStatus.STOPPING_VM, NodeMetadata.Status.PENDING).
-           put(Deployment.InstanceStatus.ROLE_STATE_UNKNOWN, NodeMetadata.Status.UNRECOGNIZED).
-           put(Deployment.InstanceStatus.UNRECOGNIZED, NodeMetadata.Status.UNRECOGNIZED).
-           build();
-
-   private static final Map<Deployment.Status, NodeMetadata.Status> STATUS_TO_NODESTATUS =
-           ImmutableMap.<Deployment.Status, NodeMetadata.Status>builder().
-           put(Deployment.Status.DELETING, NodeMetadata.Status.PENDING).
-           put(Deployment.Status.SUSPENDED_TRANSITIONING, NodeMetadata.Status.PENDING).
-           put(Deployment.Status.RUNNING_TRANSITIONING, NodeMetadata.Status.PENDING).
-           put(Deployment.Status.DEPLOYING, NodeMetadata.Status.PENDING).
-           put(Deployment.Status.STARTING, NodeMetadata.Status.PENDING).
-           put(Deployment.Status.SUSPENDED, NodeMetadata.Status.SUSPENDED).
-           put(Deployment.Status.RUNNING, NodeMetadata.Status.RUNNING).
-           put(Deployment.Status.UNRECOGNIZED, NodeMetadata.Status.UNRECOGNIZED).
-           build();
+   private static final Map<ComputeNode.Status, NodeMetadata.Status> INSTANCESTATUS_TO_NODESTATUS =
+           ImmutableMap.<ComputeNode.Status, NodeMetadata.Status>builder().
+                   put(ComputeNode.Status.GOOD, NodeMetadata.Status.RUNNING).
+                   put(ComputeNode.Status.BAD, NodeMetadata.Status.ERROR).
+                   put(ComputeNode.Status.UNRECOGNIZED, NodeMetadata.Status.UNRECOGNIZED).
+                   build();
+
+   private static final Map<Deployment.ProvisioningState, NodeMetadata.Status> STATUS_TO_NODESTATUS =
+           ImmutableMap.<Deployment.ProvisioningState, NodeMetadata.Status>builder().
+                   put(Deployment.ProvisioningState.ACCEPTED, NodeMetadata.Status.PENDING).
+                   put(Deployment.ProvisioningState.READY, NodeMetadata.Status.PENDING).
```

Comment:
How come `ACCEPTED` and `READY` are not mapped to `RUNNING` like the ASM implementation?